# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/inetutils/path-proc-net.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- inetutils-2.0/ifconfig/system/linux.c.vanilla	2021-02-06 17:16:29.261063633 +0100
+++ inetutils-2.0/ifconfig/system/linux.c	2021-02-06 17:17:23.747066444 +0100
@@ -50,6 +50,10 @@
 #include <read-file.h>
 #include <unused-parameter.h>
 
+#ifndef PATH_PROCNET_DEV
+#define PATH_PROCNET_DEV "/proc/net/dev"
+#endif
+
 #include "../ifconfig.h"
 
 
